packages/lodestar/src/chain/factory/block/body.ts

```diff
@@ -120,9 +143,14 @@ async function prepareExecutionPayload(
     parentHash = state.latestExecutionPayloadHeader.blockHash;
   }
 
-  const timestamp = computeTimeAtSlot(chain.config, state.slot, state.genesisTime);
-  const random = getRandaoMix(state, state.currentShuffling.epoch);
-  return chain.executionEngine.preparePayload(parentHash, timestamp, random, feeRecipient);
+  const timestamp = numToQuantity(computeTimeAtSlot(chain.config, state.slot, state.genesisTime));
```

Comment:
`numToQuantity` and `bytesToData` encodings should not happen at this level. Just pass regular JS values to `notifyForkchoiceUpdate` and encode in the API client code.